netebpfext/net_ebpf_ext.c

```diff
@@ -720,22 +720,29 @@ net_ebpf_ext_register_providers()
     NET_EBPF_EXT_LOG_ENTRY();
 
     status = net_ebpf_ext_xdp_register_providers();
-    if (status != STATUS_SUCCESS)
+    if (status != STATUS_SUCCESS) {
```

Comment:
I'm curious why you're mixing `status != STATUS_SUCCESS` here and `!NT_SUCCESS(status)` below.

Reply:
This was unintentional, fixed it.